src/dropdowns/dropdown.ts

```diff
@@ -0,0 +1,214 @@
+import {Directive, Input, Output, EventEmitter, HostBinding, HostListener, ElementRef, Optional, OnInit, AfterContentInit, OnDestroy, ContentChildren, QueryList} from 'angular2/core';
```

Comment:
I believe that a `component` would encapsulate better all the `class` boilerplate and would make sure that the look and feel as well as the HTML semantics, would not be broken by a user error.